src/app/collective-rewards/rewards/hooks/getTotalPotentialRewards.ts

```diff
@@ -14,7 +14,7 @@ export const useGetTotalPotentialReward = () => {
   })
 
   return {
-    data,
+    data: data || 0n,
```

Comment:
We usually handle the undefined from where it's being called.  Right now if you check the other hooks, we pass the data as it is.